.vscode/settings.json

```diff
@@ -77,5 +77,10 @@
     "data/world_americas.lua",
     "old/",
     "base/scenarios/"
+  ],
```

Comment:
These shouldn't be needed here. Make sure to reupdate your typedefs from the mod debugger extension  (click on the version picker in the bottom left status bar)